### ISSUE TITLE
Disenchanting: change from level cost change to xp cost?

### ISSUE THREAD

**Issue by Soggyshorts** [fzzyhmstrs/ai#150]
**TL;DR** could we change the config so that disenchanting costs straight XP rather than levels, please?

When taking an enchantment off of an item:

1. Take 500 xp from storage, gain 19 levels, then
2. Take an enchantment from an item (cost is 11 levels) 
**= total cost 388 xp**
or
1. Take 50 xp from storage (gain 4 levels)
2. Take 50 xp from storage (gain 3 levels)
3. Take 50 xp from storage (gain 2 levels)
4. Take 50 xp from storage (gain 2 levels)
5. Take an enchantment from an item (cost is 11 levels) 
**= total cost 187 xp**

It's not a huge deal a few times, but if you are farming enchantments then doing it the slow but efficient way 100s of times is quite tedious.

Thank for the great mod!
